web-src/src/helpers/PromptRenderer.js

```diff
@@ -54,7 +54,31 @@ export function createContentModelPrompt(contentFragmentModel) {
     + '\n```';
 }
 
-export function renderPrompt(prompt, placeholders, contentFragmentModel) {
-  const extraPrompt = contentFragmentModel ? createContentModelPrompt(contentFragmentModel) : '';
-  return removeEmptyLines(resolvePlaceholders(prompt, placeholders)) + extraPrompt;
+export function addContentFragmentContext(fragment) {
+  if (!fragment) return '';
+
+  return '\n\nOriginal content fragment for context: ```'
```

Comment:
We might consider inlining this into the Additional requirements. Just structure the list in JSON or YAML format. LLMs know both well. Something like this:
```yaml
properties:
  - name: "blah"
    description: "blah-blah"
    original_value: "blah-blah-blah"
  - name: "blah"
    description: "blah-blah"
    original_value: "blah-blah-blah"
```